clusters/app.ci/supplemental-ci-images/dynamic-plugin-sdk-buildroot.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: openshift-dynamic-plugin-sdk-buildroot
+  namespace: ci
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: openshift-dynamic-plugin-sdk-buildroot
+  namespace: ci
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
+      name: openshift-dynamic-plugin-sdk-buildroot:nodejs16
+      namespace: ci
+  source:
+    dockerfile: |
+      FROM quay.io/centos/centos:stream8
```

Comment:
@bear-redhat @vojtechszocs here is the result from the build of this dockerfile locally: 
[dynamic-plugin-sdk-buildroot-docker-build.log](https://github.com/openshift/release/files/9569600/dynamic-plugin-sdk-buildroot-docker-build.log)